packages/dnb-eufemia/src/elements/typography/__tests__/Heading.screenshot.test.ts

```diff
@@ -8,9 +8,16 @@ import {
   setupPageScreenshot,
 } from '../../../core/jest/jestSetupScreenshots'
 
-describe('Heading', () => {
+const themes = [
+  ['ui', '/uilib/elements/heading'],
+  ['sbanken', '/uilib/elements/heading'],
+  ['eiendom', '/uilib/elements/heading'],
+]
+
+describe.each(themes)('Heading for %s', (themeName, url) => {
```

Comment:
Nice! 

What you also can do, instead of having the `url` 3 times the same:

```suggestion
describe.each(['ui', 'sbanken', 'eiendom'])('Heading for %s', (themeName) => {
```

And set the `url` as an object parameter directly.